main.go

```diff
@@ -239,11 +233,6 @@ func main() {
 		setupLog.Error(err, "error remove deprecated resources from previous version")
 	}
 
-	// Apply update from legacy operator
-	if err = upgrade.UpdateFromLegacyVersion(setupClient, platform, dscApplicationsNamespace, dscMonitoringNamespace); err != nil {
```

Comment:
Is it part of this patch?

Reply:
follow up in https://github.com/red-hat-data-services/rhods-operator/pull/284